docs/talks/index.html

```diff
@@ -21,7 +21,7 @@
 -->
 
 <div class="talks" data-plugin="paginate" data-max-posts="2" data-previous=".paginate-previous--talk" data-next=".paginate-next--talk" data-pagination=".pagination--talk">
-    {% assign sorted = (site.talks | sort: 'date') | reverse %}
+    {% assign sorted = site.talks | sort: 'date' | reverse %}
```

Comment:
same as above. can we exclude these changes if they are not part of the original issue this RB is solving?

Reply:
Yes this is due to the change from redcarpet to kramdown